### DISCUSSION TITLE
How can I store a graph asset?

### DISCUSSION THREAD

**Discussion by marcilj** [dagster-io/dagster#19410] [Q&A] [answered]
I'm trying out the `graph_asset` feature and I don't understand how I'm suppose to store the `graph_asset`.

Normally I would create and asset and add my IO_Manager so the storage would get handled by default, but `graph_asset` decorator doesn't have a `io_manager_key` parameter so I'm confused on how I should handle this.

I could store it directly into a table using and `op`, but then how could the potential `downstream` asset of this `graph_asset` would know how to load the data from this `graph_asset?

I was under the impression that I `should`/`could` do something like this but I can't
```python
@op
def my_op():
    return 1

@op
def my_second_op(something):
    return something + 1

@graph_asset(io_manager_key="my_io_manager")
def my_graph_asset(): 
    return my_second_op(my_op())
```

**Comment by marcilj** [comment, marked as answer]
Found it, the Graph asset is store with the `io_manager` configured in the last Op of the Graph.

**Comment by jscheel** [comment]
@marcilj can you please explain how you did this? I can't seem to be able to define an io manager on an op.

**Reply by zyd14** [reply]
```python
@op(out=Out(io_manager_key="the_io_manager_for_the_output"))
def my_last_graph_asset_op():
  ...
```
